### PR TITLE
Home work has been done.

### PR DESCRIPTION
**Pull request by max-java**

### CODE REVIEW

File: homework/english/maksim_shelkovich/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Homework 1 task 1
+
+Hello world! this is my homework.
```
Comment (javaGuruBY) [inline homework/english/maksim_shelkovich/lecture1/task1.md:3]: This is not a homework. Plese, do HW!

- [ ]  add at least 1 task

**Review by javaGuruBY — CHANGES_REQUESTED** [review]
See notes

File: homework/english/maksim_shelkovich/lecture1/task1.md (hunk 1 above)
Comment (max-java) [reply homework/english/maksim_shelkovich/lecture1/task1.md:3]: ok

**Review by javaGuruBY — APPROVED** [review]
lgfm

File: homework/english/maksim_shelkovich/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Homework 1 task 1
+
```
Comment (javaGuruBY) [inline homework/english/maksim_shelkovich/lecture1/task1.md:2]: New review for closed request!

**Review by javaGuruBY — COMMENTED** [review]
Closed< but reviewd

**Pull request merged**